src/navigation/navigation.cpp

```diff
@@ -37,18 +37,18 @@ Navigation::Navigation(Logger& log, unsigned int axis/*=0*/)
            curr_msmt_(0),
            imu_reliable_ {{true, true, true, true}},
            nOutlierImus_(0),
-           stripe_counter_(0, 0),
-           keyence_used_(true),
-           keyence_real_(true),
-           keyence_failure_counter_(0),
            acceleration_(0, 0.),
            velocity_(0, 0.),
-           distance_(0, 0.),
-           distance_uncertainty_(0.),
-           velocity_uncertainty_(0.),
+           displacement_(0, 0.),
+           displ_unc_(0.),
+           vel_unc_(0.),
            init_time_set_(false),
+           stripe_counter_(log_, data_, displ_unc_,
+            vel_unc_, kStripeDistance),
```

Comment:
```suggestion
           stripe_counter_(log_, data_, displ_unc_, vel_unc_, kStripeDistance),
```
Why break the line? It's just 79 chars. (And if you really wanna break it, at least start the new line beneath the first argument of the previous one.)